pkgs/applications/networking/clash-verge/default.nix

```diff
@@ -48,7 +48,7 @@ stdenv.mkDerivation rec {
   '';
 
   postFixup = ''
-    ln -s ${lib.getExe clash-meta} $out/bin/clash-meta
+    ln -s ${lib.getExe mihomo} $out/bin/clash-meta
```

Comment:
Is this still called clash-meta?

Reply:
> Is this still called clash-meta?

`clash-verge` has not changed the name yet.